### PR TITLE
[control-plane-manager][terraform-manager][user-authn] relax restrict…

### PR DESCRIPTION
**Pull request by m-vasilenko**
## Description
Time units for hours and minutes can be used simultaneously in configuration:

- `idTokenTTL` in `user-authn` module configuration (this one also allows seconds)
- `webhookCacheTTL` in `control-plane-manager` module configuration
- `autoConvergerPeriod` in `terraform-manager` module configuration

## Why do we need it, and what problem does it solve?
See points 4,5 and 7 in #4199 

## What is the expected result?
Settings with time.Duration can use hours and minutes simultaneously.

## Checklist
- [x] The code is covered by unit tests.
- [x] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: control-plane-manager
type: fix
summary: Hours and minutes can be used simultaneously in the `webhookCacheTTL` module configuration field.
impact_level: default
```

```changes
section: terraform-manager
type: fix
summary: Hours and minutes can be used simultaneously in the `autoConvergerPeriod` module configuration field.
impact_level: default
```

```changes
section: user-authn
type: fix
summary: Hours, minutes and seconds can be used simultaneously in the `idTokenTTL` configuration parameter .
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @m-vasilenko. I've started the workflow for label 'e2e/run/openstack'...

File: modules/040-terraform-manager/openapi/config-values.yaml [hunk 1]
```diff
@@ -8,9 +8,11 @@ properties:
   autoConvergerPeriod:
     type: string
     default: "1h"
-    pattern: '^\d+(?:m|h)$'
+    pattern: '^([0-9]+h([0-9]+m)?|[0-9]+m)$'
```
Comment (nabokihms) [inline modules/040-terraform-manager/openapi/config-values.yaml:11]:
```suggestion
    pattern: '^([0-9]+h)?([0-9]+m)?([0-9]+s)?$'
```

File: modules/040-control-plane-manager/openapi/config-values.yaml [hunk 2]
```diff
@@ -114,9 +114,11 @@ properties:
               Authorization webhook CA.
           webhookCacheTTL:
             type: string
-            pattern: '([0-9]*(\.[0-9]*)?[hms]+)+'
+            pattern: '^([0-9]+h([0-9]+m)?|[0-9]+m)$'
```
Comment (nabokihms) [inline modules/040-control-plane-manager/openapi/config-values.yaml:117]:
```suggestion
            pattern: '^([0-9]+h)?([0-9]+m)?([0-9]+s)?$'
```

Comment (nabokihms) [reply modules/040-control-plane-manager/openapi/config-values.yaml:117]: Seconds were previously allowed for the webhook.

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @m-vasilenko. I've started the workflow for label 'e2e/run/openstack'...

**Review by nabokihms — APPROVED** [review]

**Pull request merged**